.kodiak/config.yaml

```diff
@@ -0,0 +1,28 @@
+version: 1.0
+
+snow:
+  - id: 546046 # Milo Bacom (business.adobe.com)
+
+notifications:
+  jira:
+    default:
+      project: MWPW # Mandatory
+      filters:
+        include:
+          risk_rating: R5
+      fields:
```

Comment:
Is it possible to set the Team to Magma?

Reply:
@meganthecoder Possibly. I think Jed was able to do it for `caas`. There's more documentation on the available fields here if you are interested.  https://git.corp.adobe.com/kodiak-config-examples/default-notification-config/blob/main/Example%20configs/base-config.yaml

Reply:
Found it. Added it in the next commit. Let me know if there's any other tweaks you'd like me to make.